packages/runner/src/Runner.ts

```diff
@@ -53,9 +61,10 @@ export class Runner
 
     /**
      * Dispatch/Broadcast Runner to all listeners added to the queue.
-     * @param {...any} params - optional parameters to pass to each listener
+     * @param {...any} [params] - optional parameters to pass to each listener
+     * @return {PIXI.Runner}
      */
-    emit(a0, a1, a2, a3, a4, a5, a6, a7)
+    public emit(a0?: any, a1?: any, a2?: any, a3?: any, a4?: any, a5?: any, a6?: any, a7?: any): this
```

Comment:
Why `...a` can't be used instead of `a0, a1,a2` ?
Then it can be used instead of `arguments`.

Reply:
It’s done this way to avoid an apply or call, which has less performance